class.jetpack-cli.php

```diff
@@ -559,6 +559,184 @@ public function options( $args, $assoc_args ) {
 				break;
 		}
 	}
+
+	public function sync( $args, $assoc_args ) {
+		if ( ! Jetpack::is_active() ) {
```

Comment:
I think it would be great to check if other conditions for sync are meet. See sync_allowed method in https://github.com/Automattic/jetpack/blob/master/sync/class.jetpack-sync-actions.php#L91